assertion error in script post.py

Hi,

I have the following error message when executing conda-build for recipe https://github.com/bioconda/bioconda-recipes/tree/master/recipes/parafly:

`Traceback (most recent call last):
  File "/tmp/mc2/bin/conda-build", line 6, in <module>
    sys.exit(conda_build.cli.main_build.main())
  File "/tmp/mc2/lib/python2.7/site-packages/conda_build/cli/main_build.py", line 242, in main
    execute(sys.argv[1:])
  File "/tmp/mc2/lib/python2.7/site-packages/conda_build/cli/main_build.py", line 234, in execute
    already_built=None, config=config)
  File "/tmp/mc2/lib/python2.7/site-packages/conda_build/api.py", line 77, in build
    need_source_download=need_source_download, config=config)
  File "/tmp/mc2/lib/python2.7/site-packages/conda_build/build.py", line 1099, in build_tree
    config=recipe_config)
  File "/tmp/mc2/lib/python2.7/site-packages/conda_build/build.py", line 791, in build
    croot=config.croot)
  File "/tmp/mc2/lib/python2.7/site-packages/conda_build/post.py", line 430, in post_build
    mk_relative(m, f, prefix)
  File "/tmp/mc2/lib/python2.7/site-packages/conda_build/post.py", line 389, in mk_relative
    mk_relative_linux(f, prefix=prefix, rpaths=m.get_value('build/rpaths', ['lib']))
  File "/tmp/mc2/lib/python2.7/site-packages/conda_build/post.py", line 362, in mk_relative_linux
    rel_ours, rel_stdlib, f, rpath)
AssertionError: utils.relative ./lib and relpath lib disagree for ParaFly, lib
`

Looks like the assertion `assert rel_ours == rel_stdlib` in post.py should pass but did not because of `./`
Am I right?

Loraine

--- a/conda_build/post.py
+++ b/conda_build/post.py
@@ -364,8 +364,8 @@ def mk_relative_linux(f, prefix, rpaths=('lib',)):
             # IMHO utils.relative shouldn't exist, but I am too paranoid to remove
             # it, so instead, make sure that what I think it should be replaced by
             # gives the same result and assert if not. Yeah, I am a chicken.
-            rel_ours = utils.relative(f, rpath)
-            rel_stdlib = os.path.relpath(rpath, os.path.dirname(f))
+            rel_ours = os.path.normpath(utils.relative(f, rpath))
+            rel_stdlib = os.path.normpath(os.path.relpath(rpath, os.path.dirname(f)))
             assert rel_ours == rel_stdlib, \
                 'utils.relative {0} and relpath {1} disagree for {2}, {3}'.format(
                 rel_ours, rel_stdlib, f, rpath)

